### PR TITLE
Update ggcr

### LINKED ISSUE #90: KO not authenticating against the target docker registry/repo
After successfully using KO to [build the NATS channel controller and dispatcher](https://github.com/knative/eventing-contrib/blob/master/natss/config/README.md), KO failed to deploy the image to the target registry, in this case, docker.io

`below are the execution steps to replicate this issue`
1. set `KO_DOCKER_REPO` = `pkaisharis` [account name]
1. run `ko apply -f natss/config`
1. execution results

```
2019/09/24 15:00:54 Publishing index.docker.io/pkaisharis/channel_controller-c1ecaddae73c4ef9fc778b143614b0c2:latest
2019/09/24 15:00:54 Publishing index.docker.io/pkaisharis/channel_dispatcher-c87d09956961b64d037b8b7969b36e36:latest
2019/09/24 15:00:56 error processing import paths in "natss/config/500-dispatcher.yaml": unsupported status code 401
```

Below is the docker.io auth config from `.docker/config.json`
```
{
	"auths": {
		"https://index.docker.io/v1/": {}
	},
	"HttpHeaders": {
		"User-Agent": "Docker-Client/19.03.2 (linux)"
	},
	"credsStore": "secretservice"
}
```

### PR DESCRIPTION
**Pull request by jonjohnsonjr**
Fixes #90

This picks up an auth rewrite.

### CODE REVIEW

**Comment by jonjohnsonjr** [comment]
@pksurferdad is it easy for you to test this branch to see if it resolves your issue?

File: go.mod [hunk 1]
```diff
@@ -3,47 +3,27 @@ module github.com/google/ko
 go 1.12
```
Comment (ImJasonH) [inline go.mod]: Can you mark this as generated?

Comment (jonjohnsonjr) [reply go.mod]: Sometimes it's not generated :(

Comment (ImJasonH) [reply go.mod]: That's true, sometimes `go.mod` is meaningful. But `go.sum` and `vendor/modules.txt` is AFAIK always generated, we could just skip those. (Doesn't have to be in this PR)

**Pull request merged**